src/app/common/signature/requests.ts

```diff
@@ -72,10 +72,6 @@ export async function verifySignatureRequest({
   const token = decodeToken(requestToken);
   const signature = token.payload as unknown as CommonSignaturePayload;
   const { publicKey, stxAddress } = signature;
-  if (!publicKey) {
```

Comment:
This entire fn can be deleted as its only used in the hook. `yarn lint` should fail because this counts as an unused export.

Reply:
It seems that the hook is used in 2 different parts of the app, see https://github.com/hirosystems/wallet/pull/3806#issuecomment-1576553692

Reply:
Suggest option 3